client/src/components/ProductDetail/ProductDetail.js

```diff
@@ -32,15 +50,24 @@ export default function ProductDetail({ location }) {
 
 ShopListItem.propTypes = {
   brand: PropTypes.string,
-  shoe: PropTypes.string,
+  model: PropTypes.string,
   styleId: PropTypes.string,
   media: PropTypes.array,
   shops: PropTypes.array,
 }
 
 const ProductContainer = styled.main`
-  display: grid;
-  gap: 20px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
```

Comment:
Be careful with `gap` for flexbox as it is not supported in all browsers yet. Also, you could write it with grid and save yourself one line of code 😃 
```suggestion
  display: grid;
  gap: 10px;
```